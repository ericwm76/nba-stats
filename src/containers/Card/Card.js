import React from 'react';

export const Card = ({ player }) => {
  return (
    <div>
      <h3>Name: {player.first_name} {player.last_name}</h3>
      <h3>Team: {player.team.full_name}</h3>
      <h3>Height: {player.height_feet}'{player.height_inches}"</h3>
      <h3>Weight: {player.weight_pounds} lbs</h3>
      <h3>Position: {player.position}</h3>
      <div className="card-footer">
        <button>View Stats</button>
        <button>X</button>
      </div>
    </div>
  )
}