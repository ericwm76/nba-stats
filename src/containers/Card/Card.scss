.Card {
  border: 1px solid white;
  border-radius: 3px;
  display: inline-block;
  height: 250px;
  padding: 10px;
  width: 30%;
  h3 {
    font-size: 1.5em;
    font-weight: 400;
    margin: 5px 0px;
  }
  p {
    font-size: 1.2em;
    margin: 5px 0px;
  }
  button {
    width: 40%;
  }
}