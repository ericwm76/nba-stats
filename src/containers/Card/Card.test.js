import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';


describe('Card', () => {
  let wrapper;
  const player = {
    first_name: "Lebron",
    last_name: "James",
    height_feet: 6,
    height_inches: 8,
    weight_pounds: 250,
    position: "F",
    id: 237,
    team: {
      full_name: "Los Angeles Lakers",
    }
  }
  beforeEach(() => {
    wrapper = shallow(
      <Card 
        player={player}
      />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });

});