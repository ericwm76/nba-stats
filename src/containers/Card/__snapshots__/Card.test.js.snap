// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div
  className="Card"
>
  <h3>
    Name: 
    Lebron
     
    James
  </h3>
  <p>
    Team: 
    Los Angeles Lakers
  </p>
  <p>
    Height: 
    6
    '
    8
    "
  </p>
  <p>
    Weight: 
    250
     lbs
  </p>
  <p>
    Position: 
    F
  </p>
  <div
    className="card-footer"
  >
    <Link
      to="/player/237"
    >
      <button>
        View Stats
      </button>
    </Link>
    <button>
      X
    </button>
  </div>
</div>
`;
